cheroot/ssl/builtin.py

```diff
@@ -102,12 +136,42 @@ def __init__(
         if self.ciphers is not None:
             self.context.set_ciphers(ciphers)
 
+        self._server_env = self.env_cert_dict(
+            'SSL_SERVER', self.parse_cert(certificate, private_key),
+        )
+        if self._server_env:
```

Comment:
I prefer having "reverse clauses". By doing
```suggestion
        if not self._server_env:
            return
```
you can reduce nesting and improve readability. Please do this here and below.

Reply:
done